android/src/main/kotlin/be/tramckrijte/workmanager/Extractor.kt

```diff
@@ -269,4 +278,13 @@ object Extractor {
                 }
                 .build()
     }
+
+    private fun extractPayload(call: MethodCall): String? {
+        try {
+            return call.argument<String>(REGISTER_TASK_PAYLOAD_KEY)
+        } catch (ex: Exception) {
+            Log.e(logTag, "Can't parse input data", ex)
+        }
+        return null
```

Comment:
🤔 Hmm.
Is this still relevant?

I think `call.argument<String>(REGISTER_TASK_PAYLOAD_KEY)` is enough.

Reply:
Thank you. Removed